terminator/__main__.py

```diff
@@ -6,22 +6,29 @@
 
 
 def pause():
-    time.sleep(arguments.polling_interval)
+    time.sleep(arguments.refresh_job_interval)
 
 
 def loop():
-    previous_jobs = []
+    jobs = []
+    refresh_counter = 0
 
     while True:
-        if not previous_jobs:
-            display.clear_screen()
 
-        current_jobs = jenkins.get_jobs()
+        if refresh_counter >= arguments.refresh_view_frequency:
+            job_names = jenkins.get_job_names()
+            jobs = jenkins.get_jobs(job_names)
+            refresh_counter = 0
+        elif not jobs:
+            display.clear_screen()
+            job_names = jenkins.get_job_names()
+            jobs = jenkins.get_jobs(job_names)
+        else:
+            jobs = jenkins.refresh_jobs(jobs)
 
-        if previous_jobs != current_jobs:
```

Comment:
we could put something like that back in however there was a small bug with that in the "Started X mins ago" text didnt update unless soemthing happened with another job.